src/runner_manager.py

```diff
@@ -481,18 +533,14 @@ def reconcile(self, quantity: int, resources: VirtualMachineResources) -> int:
         Returns:
             Difference between intended runners and actual runners.
         """
-        if self.config.are_metrics_enabled:
-            start_ts = time.time()
+        start_ts = time.time()
```

Comment:
Is this relevant to flake8? It seems to remove a condition.

Reply:
The conditional here makes the function have too many branching statements which flake8 wasn't happy about :( This doesn't impact the code in any way hence i've removed the conditional here.